docs/tutorials/google/calibration_api.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>people are going to complain that this is slow and slows down notebook testing. The other notebooks loosen the tolerance parameters for fitting so the optimization finishes faster. But of course we don't want to set bad defaults that people will blindly copy.</p><p> </p><p>Did @dabacon's scheme to regex replace parameters for notebook testing get merged? If so; we need to have replacement rules for this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
It did get merged, see the .tst files next to the notebooks.

Reply:
<p>Yes, <a href="https://github.com/quantumlib/Cirq/pull/4258/files#diff-c71971c397c6f998300255f2dc1e72af2e9710003679dce0f6497c1ae66c43b3R17" target="_blank">calibration_api.tst</a> reduces the cycle depths to speed up testing here.</p>